### ISSUE TITLE
[Question] Include library code in the tidal plugin for upmpdcli?

### ISSUE THREAD

**Issue by GioF71** [tamland/python-tidal#286]
Hello, would you agree to let me include your code in the tidal plugin for upmpdcli? That would simplify the installation process.
Your work is already credited of course in the documentation. The user could just skip a step where he/she is required to install a system-wide python library, so using the --break-system-packages which might sound scary.
Thank you

**Comment by GioF71** [comment]
Uhm maybe it's not a good idea looking at the dependencies I would have to include anyway...

**Comment by tehkillerbee** [comment]
Not sure I understand the issue (you are not forced to install as a system wide python library). And as you point out, you will still need to install all the related dependencies :)

**Comment by GioF71** [comment]
I was talking about the upmpdcli docker image, where I install the library system wide. In the past I tried creating a venv but I encountered a few issues, for sure I did something wrong.
Also in the "standard" upmpdcli instructions, I'd prefer to avoid suggesting to install the libraries system wide, but anyway it's a something I have in the pipeline.
About including the libraries in the code of the plugin, yes, I abandoned the idea especially because of the dependencies I would have to include as well.
I think we can close this.
Thank you!
